src/cloud/project/project-patch.md

```diff
@@ -8,7 +8,7 @@ functional_areas:
 The [{{site.data.var.mcp-prod}}](https://github.com/magento/magento-cloud-patches) package
 delivers Magento patches and hot fixes, which improve the integration of all `{{site.data.var.ee}}` versions with Cloud environments and supports quick delivery of critical fixes. The package can also deliver custom patches that you add to your project root directory.
 
-The {{ site.data.var.mcp }} package is a dependency for the {{site.data.var.ct}} package and is installed or updated when you install or update the {{ site.data.var.ct }} package version. You can also use and manage the {{ site.data.var.mcp }} as a stand-alone package for an existing {{ site.data.var.ece }} project.
+The {{ site.data.var.mcp }} package is a dependency for the {{site.data.var.ct}} package and is installed or updated when you install or update the {{ site.data.var.ct }} package version. You can also use and manage the {{ site.data.var.mcp }} as a stand-alone package for an existing {{ site.data.var.ee }} project.
```

Comment:
`ee` is Magento Commerce project, as opposed to Cloud Commerce project. Is that intentional?

Reply:
Yes-- https://devdocs.magento.com/cloud/project/project-patch.html#apply-patches-to-a-magento-enterprise-or-open-source-project